support-services/src/main/scala/com/gu/support/promotions/PromotionValidator.scala

```diff
@@ -44,7 +45,22 @@ object PromotionValidator {
         Some(ExpiredPromotion)
       else
         None
+
+    def validForAnyProductRatePlan(
```

Comment:
Checks that a product has at least one rate plan which is valid for this promotion